Source/MLX/Documentation.docc/Organization/arithmetic.md

```diff
@@ -136,17 +136,24 @@ Note: the `-` and `/` operators are not able to be linked here.
 - ``cos(_:stream:)``
 - ``cosh(_:stream:)``
 - ``divide(_:_:stream:)``
+- ``divmod(_:_:stream:)``
 - ``erf(_:stream:)``
 - ``erfInverse(_:stream:)``
 - ``exp(_:stream:)``
 - ``floor(_:stream:)``
 - ``floorDivide(_:_:stream:)``
+- ``isNaN(_:stream:)``
+- ``isInf(_:stream:)``
+- ``isPositiveInf(_:stream:)``
+- ``isNegativeInf(_:stream:)``
```

Comment:
These are `isposinf` in Python API.  Should they be e.g. `isPosInf` for consistency?

Reply:
Good question.  I just looked up the base swift names and we have:

https://developer.apple.com/documentation/swift/float/isinfinite

- isFinite / isInfinite
- isNaN

They don't distinguish between positive & negative but they do spell out the Infinite.  So maybe it should be:

- ``isPositiveInfinity()``

Naming closer to the python makes sense too, but I think this in the middle probably isn't it.

Reply:
Well I wouldn't spell out `infinity` unless we are consistent there. I since they don't have a standard for `pos`/`neg` my preference is to follow the Python, C++ api. So that would be `isPosInf` and `isNegInf`. Wdyt?

Reply:
Sounds good.  I will make the change